src/pudl/package_data/notebooks/how_to_use_pudl_out.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>This takes like 10-15 minutes to run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>